dev-util/strace/strace-4.9.ebuild

```diff
@@ -40,6 +40,9 @@ src_prepare() {
 	use static && append-ldflags -static
 
 	export ac_cv_header_libaio_h=$(usex aio)
+	
+	# Stub out the -k test since it's known to be flaky. #545812
+	sed -i '1iexit 77' tests*/strace-k.test ||die
```

Comment:
Darn fingers! I'll fix it up ASAP.
-- 
Paolo Pedroni
<paolo.Pedroni@iol.it>
Il 09/mar/2016 21:05, Mike Frysinger <notifications@github.com> ha scritto:In dev-util/strace/strace-4.9.ebuild:
> @@ -40,6 +40,9 @@ src_prepare() {
>  	use static && append-ldflags -static
>  
>  	export ac_cv_header_libaio_h=$(usex aio)
> +	
> +	# Stub out the -k test since it's known to be flaky. #545812
> +	sed -i '1iexit 77' tests*/strace-k.test ||die

needs to be a space before the "die"

—Reply to this email directly or view it on GitHub.